src/utils/hooks/useLoadReview.tsx

```diff
@@ -15,8 +20,10 @@ interface UseLoadReviewProps {
 const useLoadReview = ({ reviewId, serialNumber }: UseLoadReviewProps) => {
   const [applicationName, setApplicationName] = useState<string>('')
   const [reviewSections, setReviewSections] = useState<SectionStructure>()
-  const [loading, setLoading] = useState(true)
-  const [error, setError] = useState('')
+  const [reviewStatus, setReviewStatus] = useState<ReviewStatus>()
+  const [isReviewLoaded, setIsReviewLoaded] = useState(false)
+  const [reviewError, setReviewError] = useState<string>()
+  const [checkTrigger, setCheckTrigger] = useState(false)
```

Comment:
Same as the comment I made in `useLoadApplication`, depending on what you end up doing.

Reply:
Done